app/i18n/de/conf.php

```diff
@@ -28,7 +28,7 @@
 			'display_authors' => 'Autoren',
 			'entry' => 'Artikel-Symbole',
 			'publication_date' => 'Datum der Veröffentlichung',
-			'related_tags' => 'Verwandte Tags',
+			'related_tags' => 'Artikel-Labels',
```

Comment:
article tag as well

Reply:
are you sure about this? I am refering to Settings Display -> in german, this looks like this right now:
![2021-02-14_15h46_24](https://user-images.githubusercontent.com/37835829/107879893-d5469d00-6edb-11eb-9835-7af8d62741b3.png)

So if I check this for "bottom line", it shows "my labels" in the bottom line. therefore, this seem to relate to labels?

Reply:
Labels are added by the user and tags come from the feed. Are they both toggled by the same setting?

Reply:
Update: ok, I don't get it. Even if I "uncheck" the "verwandte tags" (related tags) option in the bottom line (fußzeile), the "my labels" is showing in the bottom line in the main view. so this setting does not seem to have any effect at all.

Reply:
This toggle show article tags (from feed) in the article itself.
Labels are user generated. You cannot turn off labels. But you can turn off tags.

Reply:
No, that's what I was (more or less) expecting to happen. That means you can't turn them off in the display there, although one might wonder why not.

For tags you need an article that actually has some. They come from the feed.
![Screenshot_2021-02-14_15-53-15](https://user-images.githubusercontent.com/202757/107880082-dd530c80-6edc-11eb-8d08-44deb7047e80.png)

Reply:
oh. ok, then I mixed it up. I was not aware that you cannot turn of labels, sorry. 
seems I simply do not have any feeds which support article tags then.

Reply:
You explain it better than I was :)
I was trying to say the same thing but in the worst possible way